src/components/Box.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {YogaNode} from 'yoga-layout-prebuilt';
+
+interface BoxProps {
+	readonly width?: number | string;
+	readonly height?: number | string;
+	readonly minWidth?: number;
+	readonly minHeight?: number;
+	readonly paddingTop?: number;
+	readonly paddingBottom?: number;
+	readonly paddingLeft?: number;
+	readonly paddingRight?: number;
+	readonly paddingX?: number;
+	readonly paddingY?: number;
+	readonly padding?: number;
+	readonly marginTop?: number;
+	readonly marginBottom?: number;
+	readonly marginLeft?: number;
+	readonly marginRight?: number;
+	readonly marginX?: number;
+	readonly marginY?: number;
+	readonly margin?: number;
+	readonly flexGrow?: number;
+	readonly flexShrink?: number;
+	readonly flexDirection?: 'row' | 'row-reverse' | 'column' | 'column-reverse';
+	readonly flexBasis?: string | number;
+	readonly alignItems?: 'flex-start' | 'center' | 'flex-end';
+	readonly justifyContent?:
+	| 'flex-start'
+	| 'center'
+	| 'flex-end'
+	| 'space-between'
+	| 'space-around';
```

Comment:
```suggestion
	readonly justifyContent?:
		| 'flex-start'
		| 'center'
		| 'flex-end'
		| 'space-between'
		| 'space-around';
```

Reply:
I just realized that this is caused by `xo --fix`. Is this a bug?

Reply:
Definitely a bug in ESLint-plugin-react or our config for it. Most likely the former.

Reply:
This is formatted by `@typescript-eslint/indent`. I get `Expected indentation of 1 tab but found 2.` when I use your suggested format. I'm going to clone `typescript-eslint`package to see if I can replicate this in their tests.

Reply:
I believe this is the issue that we're experiencing https://github.com/typescript-eslint/typescript-eslint/issues/880